resources/tests/type_checker_tests/MaybeDroppedResource.obs

```diff
@@ -12,12 +12,12 @@ main resource contract Wallet {
 	 state Empty {
 	 }
 
-	 transaction loseMoney() available in Full, Empty {
+	 transaction loseMoney(Wallet@(Full|Empty) this) {
 	       // Error -- maybe losing Money!
```

Comment:
This is actually different and the compiler should now give an additional error: without the >> clause, the compiler should enforce that the state cannot change within the transaction.